src/components/organisms/ConversationWindow.vue

```diff
@@ -152,6 +152,16 @@ export default {
       tabbable.value = -1;
     }
 
+    function scrollToBottom() {
+      const list = document.querySelector("#conversationWindow");
```

Comment:
should use `document.getElementById("conversationWindow")`

Reply:
Great catch! Would we have more than one dynamic convo window in the dom? If there were two chat messages per say

Reply:
SOrry - chat windows - not chat messages

Reply:
We use v-if for switching the chat window/message lists, so we'll always have at most one element with the conversationWindow id or the messageList id. Related to [this PR](https://github.com/Toad-Labs/vc-frontend-static/pull/153) where I used the fact that there's either one of those ids to focus on either window 
`const contentWindow = document.getElementById("conversationWindow") ||
          document.getElementById("messageList");`

Reply:
Thanks for the clarification

Benoit - I agree with Tisham that the document.getElementById is the better way to go

Reply:
How about what I just pushed? I am using the already declared ref.

Reply:
Oh lol this has already been addressed in the latest commit

Reply:
Yup, that works better lol, good catch!

Reply:
I want to hit the resolve conversation button but I think Tisham or you should do the honours